requirements.txt

```diff
@@ -1,25 +1,4 @@
 aiidalab==21.06.0
-aiidalab-widgets-base~=1.0.0b18
-aiida-core[atomic_tools,docs,rest]~=1.6
-aiida-bigdft~=0.2
-aiida-codtools~=2.0
-aiida-cp2k~=1.3
-aiida-crystal17~=0.11
-aiida-ddec~=1.0
-aiida-diff~=1.2
-aiida-fleur~=1.1
-aiida-lammps~=0.8
-aiida-lsmo~=1.0
-aiida-nanotech-empa~=0.2
-aiida-optimize~=0.4
-aiida-qeq~=1.0.0a1
-aiida-quantumespresso~=3.4
-aiida-raspa~=1.2
-aiida-tbmodels~=0.3
-aiida-vasp~=2.1
-aiida-wannier90~=2.0
-aiida-yambo~=1.3
-aiida-zeopp~=1.1
```

Comment:
I don't think we should install any of the dependencies left after this line. They should be dependencies of AWB and the QE app, no?

Reply:
Well, yes, let's remove it and test the apps. In case things are missing - we will put them as the dependencies of the apps.

Reply:
I changed my mind. If we do that right now - we end up having a non-functioning image. I propose that we distribute the remaining dependencies in a separate PR.

Reply:
Why would it be non-functioning?

Reply:
because dependencies like markdown are not set anywhere.

Reply:
we just need to distribute the remaining dependencies to the appropriate packages.